src/device-registry/utils/create-airqloud.js

```diff
@@ -17,7 +17,12 @@ const logger = log4js.getLogger(
 const createLocationUtil = require("./create-location");
 const geolib = require("geolib");
 const httpStatus = require("http-status");
-const { kafkaProducer } = require("../config/kafkajs");
+
+const { Kafka } = require("kafkajs");
```

Comment:
Addressing a runtime for Kafka producer where things were undefined.
So brought all the configurations into each respective util to prevent this cyclic dependency runtime error.

Repetition of code but prevent this bug from happening ever again, hopefully!

@BenjaminSsempala @NoahNsimbe